components/Chip/Chip.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
Sorry I forgot to bring this up the other day, but there is already a component for this Called SparcPill. Could you remove this component and use that one instead? It looks to be identical. It looks like we are using it for the datasets results, I must have forgot to use it when making the about page refactor